<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkiDataSourceWidget</class>
 <widget class="QWidget" name="SkiDataSourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SkiDataView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="dataTreeWidget">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openPushButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionLoad_All">
   <property name="text">
    <string>Load All</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="text">
    <string>Unload</string>
   </property>
  </action>
  <action name="actionUnload_All">
   <property name="text">
    <string>Unload All</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionRemove_All">
   <property name="text">
    <string>Remove All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imuCostumeIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SkiDataSourceWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onCustomContextMenuRequest(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_All</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onLoadAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_All</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onRemoveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnload</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onUnload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnload_All</sender>
   <signal>triggered()</signal>
   <receiver>SkiDataSourceWidget</receiver>
   <slot>onUnloadAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpen()</slot>
  <slot>onCustomContextMenuRequest(QPoint)</slot>
  <slot>onLoad()</slot>
  <slot>onLoadAll()</slot>
  <slot>onUnload()</slot>
  <slot>onUnloadAll()</slot>
  <slot>onRemove()</slot>
  <slot>onRemoveAll()</slot>
 </slots>
</ui>
